list2.py

```diff
@@ -35,12 +35,17 @@ def remove_adjacent(nums):
 # The solution should work in "linear" time, making a single pass of both lists.
 # Hint: Don't use `sort` or `sorted` -- they are not linear time.
 def linear_merge(list1, list2):
-    """Your code goes here.  Edit this docstring."""
-    return
-
+    list = list1 + list2
```

Comment:
Watch out here.  You are using python's built-in 'list' keyword as a variable, and you are changing the functionality by doing this ..

Reply:
OMG! that's why 'list' was always on green lol.
I'll change it right now. Thank you!